doc/man3/SSL_tick.pod

```diff
@@ -0,0 +1,90 @@
+=pod
+
+=head1 NAME
+
+SSL_tick - advance asynchronous state machine and perform network I/O
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_tick(SSL *ssl);
+
+=head1 DESCRIPTION
+
+SSL_tick() performs any internal processing which is due on a SSL object. The
+exact operations performed by SSL_tick() vary depending on what kind of protocol
+is being used with the given SSL object. For example, SSL_tick() may handle
+timeout events which have become due, or may attempt, to the extent currently
+possible, to perform network I/O operations on one of the BIOs underlying the
+SSL object.
+
+The primary use case for SSL_tick() is to allow an application which uses
+OpenSSL in nonblocking mode to give OpenSSL an opportunity to handle timer
+events, or to respond to the availability of new data to be read from an
+underlying BIO, or to respond to the opportunity to write pending data to an
+underlying BIO.
+
+SSL_tick() can be used only with the following types of SSL object:
+
+=over 4
+
+=item DTLS SSL objects
+
+Using SSL_tick() on an SSL object being used with a DTLS method allows timeout
+events to be handled properly. This is equivalent to a call to
+DTLSv1_handle_timeout(). Since SSL_tick() handles a superset of the use cases of
+DTLSv1_handle_timeout(), it should be preferred for new applications which do
+not require support for OpenSSL 3.1 or older.
```

Comment:
Should we deprecate DTLSv1_handle_timeout() I wonder?

Reply:
That would seem reasonable